ramls/loan.json

```diff
@@ -152,6 +152,10 @@
               "type": "string"
             }
           }
+        },
+        "effectiveCallNumberComponents": {
```

Comment:
Personally, I wouldn't use `effective` here. I think the derivation of effective call number is a responsibility of the inventory context.

From the perspective of circulation and it's clients, these are merely properties of the item.

Admittedly, that is a trade off, if the domain terminology uses `effective` as folks might find it easier to follow if that is used all the way through the system.

@bohdan-suprun @mreno-EBSCO @rladdusaw What do you think?

Reply:
@marcjohnson-kint I agree with dropping "effective" here.